content/en/cosign/overview.md

```diff
@@ -63,24 +63,44 @@ $ cosign verify-blob <file> --bundle cosign.bundle --certificate-identity=name@e
 
 To verify, Cosign queries the transparency log (Rekor) to compare the public key bound to the certificate, and checks the timestamp on the signature against the artifact’s entry in the transparency log. The signature is valid if its timestamp falls within the small window of time that the key pair and certificate issued by the certificate authority were valid.
 
-### Working with containers
+## Example: Working with containers
 
-Signing and verifying a container is similar to working with blobs. The Cosign command to sign a container image is:
+> NOTE: In this example, we will create a container using [ttl.sh](https://ttl.sh/).  It offers free, short-lived (i.e., hours), anonymous container image hosting if you just want to try out signing and verifying commands.  
+
+To use `ttl.sh` and [crane](https://github.com/google/go-containerregistry/tree/main/cmd/crane) to prepare the image that we want to sign. Run the following:
+
+```
+$ SRC_IMAGE=busybox
+$ SRC_DIGEST=$(crane digest busybox)
+$ IMAGE_URI=ttl.sh/$(uuidgen | head -c 8 | tr 'A-Z' 'a-z')
+$ crane cp $SRC_IMAGE@$SRC_DIGEST $IMAGE_URI:1h
+$ IMAGE_URI_DIGEST=$IMAGE_URI@$SRC_DIGEST
+```
+
+### Keyless signing of a container
+
+The following code signs the created container image.  The command to sign container images is `cosign sign <image URI>`.  Note that for containers, there is no bundle as there is with blobs, as the signature and certificate are attached directly to the container:
 
 ```
-$ cosign sign <image URI>
+$ cosign sign $IMAGE_URI_DIGEST
 ```
 
-This works the same as signing a blob, but the signature and certificate are attached as container metadata.
+### Keyless verifying of a container
 
-To verify a signed container image, use the following command:
+This works the similar to verifying a blob, but there is no need to place the certificate and signature on the verify command. To verify a signed container image, use the following command:
 
 ```
 $ cosign verify <image URI> --certificate-identity=name@example.com
```

Comment:
Consistency here?

```suggestion
$ cosign verify $IMAGE_URI_DIGEST --certificate-identity=name@example.com
```

Reply:
Changing it like this would imply to readers that they should type $IMAGE_URI_DIGEST in the command, and that's not correct.